package com.cydeo.test.day6_stale_exceptions;

import org.testng.annotations.Test;

public class TestNg_intro {

    @Test
    public void test(){
        System.out.println("Testing is starting");
    }
    @Test
    public void test2(){
        System.out.println("Testing is going on");
    }
}
